### DISCUSSION TITLE
Change player faction to BAF

### DISCUSSION THREAD

**Discussion by ross3022** [Vdauphin/HeartsAndMinds#1520] [Question and answer]
Hey so I'm wanting to change the player faction to British forces.
I've added the vehicles into the layers on eden.
How do I change the soldiers etc from US to BAF?

**Comment by Vdauphin** [comment]
Hello,
> How do I change the soldiers etc from US to BAF?

By default, H&M generate loadout dynamically with player slot and day/night
You can tweak it here: https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/define_mod.sqf#L51-L74

If you want to create your loadout by hand, first you need to disable that feature in mission parameters: https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/param.hpp#L434-L439
Then you can customize each soldier in your mission.sqm

Cheers
